bank2ynab.conf

```diff
@@ -13,20 +13,21 @@ Source CSV Delimiter = ,
 Header Rows = 1
 Footer Rows = 0
 Input Columns = Date,Payee,Outflow,Inflow,Running Balance
-# (see https://docs.python.org/2/library/datetime.html#id1 for date format strings)
+# (see https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes for date format strings)
 Date Format =
 Inflow or Outflow Indicator =
 # Output file formatting
 Output Columns = Date,Payee,Category,Memo,Outflow,Inflow
 Output Filename Prefix = fixed_
 Use Payee for Memo = True
 # Post-processing
-Delete Source File = True
+Delete Source File = False
 # Plugins
 Plugin =
 # YNAB API Stuff - don't modify values here, modify in user_configuration.conf
-YNAB API Access Token =
-YNAB Account ID =
+#ffc24c41e613dec1e9eb72bfac15837c7b3014635963f952a01a44d86dac393c
```

Comment:
Is this your personal API key??

Reply:
it was accidentially commited and revoked some time ago.

Reply:
https://github.com/bank2ynab/bank2ynab/pull/386#commitcomment-47634433